src/api/entities/entityTypes.ts

```diff
@@ -0,0 +1,774 @@
+import BigNumber from 'bignumber.js';
```

Comment:
thought about putting these below the rexports in `types` file? Seems a bit redundant to have two type files beside each other.